h5py/_hl/group.py

```diff
@@ -667,6 +673,63 @@ def proxy(name):
                 return func(name, self[name])
             return h5o.visit(self.id, proxy)
 
+    def visit_links(self, func):
+        """ Recursively visit all names in this group and subgroups, including links.
```

Comment:
Technically, all names in a group are links - but normally they're hard links, pointing to an object rather than a name.

IDK how important it is to make this clear, but it does lead to another significant difference between these methods & the existing (object-based) ones. If you have two hardlinks to the same object, the link-based functions will see both of them, but the object-based functions will only see one - whichever one they reach first.

Reply:
Understood. I pushed a commit to mention this in the docstring.